engine/src/main/java/com/google/android/fhir/search/SearchDsl.kt

```diff
@@ -62,6 +89,13 @@ data class StringFilter(
 @SearchDslMarker
 data class ReferenceFilter(val parameter: ReferenceClientParam?, var value: String? = null)
 
+@SearchDslMarker
+data class TokenFilter(
+  val parameter: TokenClientParam?,
+  var system: String? = null,
```

Comment:
Use `uri` to match the definition here https://www.hl7.org/fhir/STU3/search.html#token?

```suggestion
  var uri: String? = null,
```

Reply:
Renamed variable to uri